configure.ac

```diff
@@ -35,25 +35,6 @@ LT_INIT
 
 dnl Checks for libraries.
 
-dnl Checks for header files.
-AC_CHECK_HEADERS(crypt.h utmp.h \
-	termio.h sgtty.h sys/ioctl.h paths.h \
-	sys/capability.h sys/random.h \
-	gshadow.h lastlog.h rpc/key_prot.h acl/libacl.h \
-	attr/libattr.h attr/error_context.h)
-
-dnl shadow now uses the libc's shadow implementation
-AC_CHECK_HEADER([shadow.h],,[AC_MSG_ERROR([You need a libc with shadow.h])])
-
-AC_CHECK_FUNCS(arc4random_buf futimes \
-	getentropy getrandom getspnam getusershell \
-	initgroups lckpwdf lutimes \
-	setgroups updwtmp updwtmpx innetgr \
-	getspnam_r \
-	rpmatch \
-	memset_explicit explicit_bzero stpecpy stpeprintf)
-AC_SYS_LARGEFILE
-
 dnl Checks for typedefs, structures, and compiler characteristics.
 
 AC_CHECK_MEMBERS([struct utmp.ut_type,
```

Comment:
i think some of these tests below could be affected by which headers are detected.  worth double checking config.log to see what the generated tests are with the default set of header includes.